assets/js/phoenix_live_view.js

```diff
@@ -407,7 +408,14 @@ export class LiveSocket {
   }
 
   bindTopLevelEvents(){
-    this.bindClicks()
+    if ("ontouchstart" in window) {
+      this.bindTouchstarts()
+      this.bindTouchmoves()
+      this.bindClicks("touchend")
```

Comment:
Is there an issue of not using the "click" event in iOS or other touch enabled device?  Or is this a sufficient replacement?

Reply:
I tried using "click", but it only worked when the element is an `<a>`. I found this solution at #168, but one of my use cases is to highlight paragraphs, and I don't want to convert every `<p>` to an `<a>` (although it is doable).

I also tried [`cursor:pointer`](https://stackoverflow.com/questions/3025348/how-do-i-use-jquery-for-click-event-in-iphone-web-application/4910962#4910962) that you listed in your comment, it doesn't work for me when the element isn't an `<a>`. My guess is it works only if the EventListener is binded to *the element*, but live_view binds the click events to `window`. but I am not so sure if it is the reason. 

In another link you listed in the comment, the doc explains how mobile-safari handles event, that is super helpful for me to reason why iOS is having this issue - because only `element.onClick` is fired, while the `window.onClick` isn't.

I tested my app on MobileChrome on a samsung S8, it works as expected. But I can't prove it is a sufficient replacement for other touch enabled devices.